docs/tutorial/numbers.md

```diff
@@ -28,4 +28,8 @@ x =  35e3
 y =  12e4  
 z = -87.7e100  
 ```
-
+Both integers and floating point numbers are asumed to be true if queried ,unless the have an e,
+**SO:**
+[This works](https://unv.vercel.app/playground#import%20print%20from%20'standard'%0A%0Aif%206.457%0A%20%20%20%20print('Hello%20World!')%0A%23%20keep%20editing%20for%20live%20results%0A)
+**BUT**
+[This doesn't](https://unv.vercel.app/playground#import%20print%20from%20'standard'%0A%0Aif%206e.457%0A%20%20%20%20print('Hello%20World!')%0A%23%20keep%20editing%20for%20live%20results%0A)
```

Comment:
```suggestion
Both integers and floating point numbers except `0` are assumed to be `true` if queried.
```py
print(boolean(234)) # => true
print(boolean(0)) # => false
```py
```
your code is syntaxially incorrect. `6e.457` is wrong.

> remove last word `py`. it's added to prevent a bug in GitHub comments.

Reply:
Will do! Your explanation makes more sense anyway.